metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/WorkflowExecutionFactory.java

```diff
@@ -254,8 +254,7 @@ private void setupXsltIdForPluginMetadata(Dataset dataset,
     if (xsltObject != null && StringUtils.isNotEmpty(xsltObject.getXslt())) {
       pluginMetadata.setXsltId(xsltObject.getId().toString());
     }
-    //DatasetName in Transformation should be a concatenation datasetId_datasetName
-    pluginMetadata.setDatasetName(dataset.getDatasetId() + "_" + dataset.getDatasetName());
+    pluginMetadata.setDatasetName(dataset.getDatasetName());
```

Comment:
We are sure that in eCloud this parameter is ONLY used as input for our validation service, right?